content/posts/0.22-release.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Pyodide 0.22.0 release"
+date: 2022-12-27 # TODO: update date before merging
+tags: ["release", "announcement"]
+author: "Gyeongjae Choi and Hood Chatham on behalf of the Pyodide team"
+showToc: true
+TocOpen: false
+hidemeta: false
+comments: false
+# description: ""
+disableHLJS: true # to disable highlightjs
+disableShare: false
+hideSummary: false
+searchHidden: true
+ShowReadingTime: true
+ShowBreadCrumbs: true
+ShowPostNavLinks: true
+---
+
+Pyodide 0.22 is a major release focused on improving the foreign function
+interface between Python and JavaScript. We also improved the systems for
+building, loading, and testing packages. In particular, we added support for
+creating virtual environments that execute code using node Pyodide.
```

Comment:
`node Pyodide` doesn't seem very clear to me. Maybe something like `Pyodide and Node.js` instead?

Reply:
Agreed that the phrasing is not good. Maybe "Pyodide in node.js"? How do we say it in the dosc?

Reply:
"Pyodide in Node.js" sounds good to me. I wasn't able to find similar wording in docs though.

Reply:
Fixed.